src/main/resources/db/migration/0300/V303__UserInternalInterests.sql

```diff
@@ -0,0 +1,16 @@
+-- Create user_internal_interests table
+CREATE TABLE accelerator.user_internal_interests (
+    user_id BIGINT NOT NULL REFERENCES users,
+    internal_interest_id INTEGER NOT NULL REFERENCES accelerator.internal_interests ON DELETE CASCADE,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+
+PRIMARY KEY (user_id, internal_interest_id)
+);
+
+-- Move data from user_deliverable_categories  to user_internal_interests
+INSERT INTO accelerator.user_internal_interests (user_id, internal_interest_id, created_by, created_time)
+SELECT user_id, deliverable_category_id as internal_interest_id, created_by, created_time
```

Comment:
You shouldn't need the `as internal_interest_id` here (the values are positional, not named).